node/parallel/src/rpc.rs

```diff
@@ -0,0 +1,63 @@
+#![warn(missing_docs)]
```

Comment:
I think we should modify https://github.com/parallel-finance/parallel/blob/master/node/parallel/src/service.rs#L323 instead

Reply:
Fixed